packages/components/src/components/SquiggleViewer/ExpressionViewer.tsx

```diff
@@ -67,7 +77,7 @@ export const ExpressionViewer: React.FC<Props> = ({
     );
   }
   switch (expression.tag) {
-    case "number":
+    case SquiggleValueTag.SvtNumber:
```

Comment:
I'm kind of surprised this works. Guess it's been a while since I've been doing heavy JS.